bedrock/pocket/templates/pocket/firefox/new-tab-learn-more.html

```diff
@@ -53,39 +53,40 @@ <h2 class="new-tab-section-header">Finding the hidden gems. Respecting your priv
+    <h2 class="new-tab-section-header">{{ ftl('pocket-new-tab-fuel') }}</h2>
+    <p class="new-tab-section-body">{{ ftl('pocket-new-tab-check-out', explore_url='https://getpocket.com/explore',
+    hits_signup_url='https://getpocket.com/explore/pocket-hits-signup') }}</p>
   {% endcall %}
   {% call split(
   block_class='mzp-l-split-center-on-sm-md mzp-t-content-xl mzp-t-split-nospace new-tab-section',
   media_class='mzp-l-split-h-center',
   image_url='img/pocket/new-tab/lounge.svg',
   )%}
-    <h2 class="new-tab-section-header">Save in Pocket & read on your own time.</h2>
-    <p class="new-tab-section-body">Built right into Firefox, Pocket also lets you save stories and come back to them when you’re ready.</p>
-    <p class="new-tab-section-body"><a href="https://getpocket.com/firefox_learnmore">Activate your account</a>, and then the next time you see an article you want to save, click the Pocket icon in the toolbar.</p>
+    <h2 class="new-tab-section-header">{{ ftl('pocket-new-tab-save-in') }}</h2>
+    <p class="new-tab-section-body">{{ ftl('pocket-new-tab-built-right') }}</p>
+    <p class="new-tab-section-body">{{ ftl('pocket-new-tab-activate-your-account', firefox_learn_url='https://getpocket.com/firefox_learnmore') }}</p>
   {% endcall %}
 </section>
 <section class="mzp-l-content">
   <img class="details-img" src="{{ static('img/pocket/new-tab/sign.svg') }}" />
   <div>
-    <h2 class="new-tab-section-header">You make the call.</h2>
+    <h2 class="new-tab-section-header">{{ ftl('pocket-new-tab-you-make') }}</h2>
     <div class="helpful-links-wrapper">
-      <p class="new-tab-section-body">It’s easy to turn off Pocket’s recommendations and remove them from your new tab page. But remember: if you don’t like today’s stories, there’s always tomorrow!</p>
+      <p class="new-tab-section-body">{{ ftl('pocket-new-tab-its-easy', dismiss_url='https://help.getpocket.com/article/1142-firefox-new-tab-recommendations-faq#turn-off') }}</p>
       <ul class="new-tab-helpful-links">
-        <li><a href="https://help.getpocket.com/article/1142-firefox-new-tab-recommendations-faq">Frequently asked questions</a></li>
-        <li><a href="https://getpocket.com/privacy">Privacy Policy</a></li>
-        <li><a href="https://getpocket.com/tos">Terms of service</a></li>
-        <li><a href="https://getpocket.com/sponsor">Sponsorship opportunities</a></li>
+        <li><a href="https://help.getpocket.com/article/1142-firefox-new-tab-recommendations-faq">{{ ftl('pocket-new-tab-faq') }}</a></li>
+        <li><a href="https://getpocket.com/privacy">{{ ftl('pocket-new-tab-privacy-policy') }}</a></li>
+        <li><a href="https://getpocket.com/tos">{{ ftl('pocket-new-tab-tos') }}</a></li>
```

Comment:
Suggestion (non-blocking):
```suggestion
        <li><a href="{{ url('pocket.tos') }}">{{ ftl('pocket-new-tab-tos') }}</a></li>
```